src/content/en/tools/workbox/guides/migrations/migrate-from-v5.md

```diff
@@ -0,0 +1,300 @@
+
+This guide is focused on breaking changes introduced in Workbox v6, with examples of what changes
+you'd need to make when upgrading from Workbox v5.
+
+## Breaking Changes
+
+### workbox-core
+
+The `skipWaiting()` method in `workbox-core` will no longer add in an `install` handler and is
+equivalent to just calling `self.skipWaiting()`. 
+
+From now on, use `self.skipWaiting()` instead since `skipWaiting()` will likely be removed in
+Workbox v7.
+
+### workbox-precaching
+
+- Cross-origin HTML documents for URLs that correspond to an HTTP redirect can no longer be used to
+  satisfy a navigation request with `workbox-precaching`. This scenario is generally uncommon.
+
+- The `fbclid` URL query parameter is now ignored by `workbox-precaching` when looking up a
+  precached response for a given request.
+
+### workbox-routing
+
+The `setDefaultHandler()` method now takes an optional second parameter corresponding to the HTTP
+method that it applies to, defaulting to `'GET'`. 
+
+- If you use `setDefaultHandler()` and all your requests are `GET`, then no changes need to be made.
+
+- If have any requests that are not `GET` (`POST`, `PUT`, etc...), `setDefaultHandler()` will no
+  longer cause those requests to match.
+
+### Build Configuration
+
+The `mode` option for the `getManifest` and `injectManifest` modes in `workbox-build` and
+`workbox-cli` was not intended to be supported and has been removed. This does not apply to
+`workbox-webpack-plugin`, which does support `mode` in its `InjectManifest` plugin.
+
+### Build Tools Require Node.js v10 or Higher
+
+Node.js versions prior to v10 are no longer supported for `workbox-webpack-plugin`, `workbox-build`,
+or `workbox-cli`. Please update your Node.js runtime to a [supported
+version](https://nodejs.org/en/about/releases/).
+
+## New Improvements
+
+### workbox-strategies
+
+Workbox v6 introduces a new way for third-party developers to define their own Workbox strategies.
+This ensures third-party developers have the capability to extend Workbox in ways that fully meet
+their needs.
+
+#### New Strategy base class
+
+In v6, all Workbox strategy classes must extend the new `Strategy` base class. All of the built-in
+strategies have been rewritten to support this. 
+
+The `Strategy` base class is responsible for two primary things:
+
+- Invoking plugin lifecycle callbacks common to all strategy handlers (e.g. when they start,
+  respond, and end).
+- Creating a "handler" instance, that can manage state for each individual request a strategy is
+  handling.
+
+#### New "handler" class
+
+We previously had internal modules call `fetchWrapper` and `cacheWrapper`, which (as their name
+implies) wrap the various fetch and cache APIs with hooks into their lifecycle. This is the
+mechanism that currently allows plugins to work, but it's not exposed to developers.
+
+The new "handler" class, `StrategyHandler`, will expose these methods so custom strategies can call
```

Comment:
@jeffposnick release docs mention naming of `StrategyHandler` is in a proposal - so let me know if the name happens to change